clouddriver-ecs/src/integration/resources/testoperations/createServerGroup-input-EC2-targetGroupMappings-existingService.json

```diff
@@ -1,6 +1,6 @@
 {
   "account": "ecs-account",
-  "application": "ecs",
+  "application": "createService",
```

Comment:
did we need to change the app name? if not i'd rather we kept it consistent (ok with other stuff with an app prefix having this)

Reply:
@allisaurus  So apparently, moniker considers 1st part of service name as app name.  So if you see the service name which is "createService-integInputEC2TgMappingsExistingServiceStack-v000" and 1st part of it before hyphen "-" is createService. So, To avoid failing one of the conditions where we check moniker.app() equals descriptions.app() I changed the application name as createService.  Does that make sense?

Reply:
yes! I guess i'm saying that i'm OK with prefixing the server group name with "ecs" instead, then, since this matches the other tests and what users will more often be doing.

Reply:
Okay. Got your point.